soc/xtensa/intel_adsp/tools/cavstool_client.py

```diff
@@ -89,8 +126,8 @@ def main():
             client.send_cmd(CMD_LOG_START)
         except KeyboardInterrupt:
             pass
-        finally:
-            cleanup()
+        #finally:
+        #    cleanup()
```

Comment:
Why this commented out code?

Reply:
Originally I want to disable in the second commit then remove it in the third commit. Thanks for pointing this out. Got it fixed.